### PR TITLE
DDCNL-1339: Updated sbt and Scala, considering Akka

### PR DESCRIPTION
**Pull request by Christian-QA**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by struong — COMMENTED** [review]
Ensure you run the integration tests to update the headers there as well

File: project/plugins.sbt [hunk 1]
```diff
@@ -8,16 +8,16 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
...
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.19.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.19.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
```
Comment (struong) [inline project/plugins.sbt:21]: this can now be removed

File: project/plugins.sbt [hunk 2]
```diff
@@ -8,16 +8,16 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
...
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.3")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.5.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.2.0")
```
Comment (struong) [inline project/plugins.sbt:17]: This has been downgraded?

**Review by struong — COMMENTED** [review]
Ensure you run the integration tests to update the headers there as well

File: project/plugins.sbt (hunk 1 above)
Comment (Christian-QA) [reply project/plugins.sbt:21]: Done

File: project/plugins.sbt (hunk 2 above)
Comment (Christian-QA) [reply project/plugins.sbt:17]: Done

**Review by struong — APPROVED** [review]

**Pull request merged**